aea/configurations/manager.py

```diff
@@ -244,7 +244,9 @@ def handle_dotted_path(
         )
 
         # find path to the resource directory
-        path_to_resource_directory = Path(".") / resource_type_plural / resource_name
+        path_to_resource_directory = (
+            aea_project_path / resource_type_plural / resource_name
+        )
```

Comment:
I think this is more accurate since we have 

https://github.com/valory-xyz/open-aea/blob/b831e33696a54ec0207b5f08e08213c0e1343d3b/aea/configurations/manager.py#L145

Reply:
yes, but `resource_type_plural` `resource_name` are of type string so it won't work with "."

suggest:
`Path(aea_project_path, resource_type_plural, resource_name)`
or
`Path(aea_project_path) / resource_type_plural / resource_name)`
which should work in all combinations of `str` and `Path`

Reply:
https://github.com/valory-xyz/open-aea/blob/b831e33696a54ec0207b5f08e08213c0e1343d3b/aea/configurations/manager.py#L169